fiasko_bro/validator_helpers.py

```diff
@@ -1,11 +1,11 @@
-from functools import wraps
+def ensure_repo_tokens_mutually_exclusive(**kwargs):
+    if 'validator_token' in kwargs and 'validator_tokens' in kwargs:
+        raise ValueError("Please specify either 'token' or 'tokens'")
 
 
-def tokenized_validator(token):
-    def validator_decorator(func):
-        @wraps(func)
-        def func_wrapper(*args, **kwargs):
-            if token == kwargs.get('validator_token'):
-                return func(*args, **kwargs)
-        return func_wrapper
-    return validator_decorator
+def if_any(tokens, repo_tokens):
+    return any(token for token in tokens if token in repo_tokens)
```

Comment:
Now you can write `any(token in repo_tokens for token in tokens)`. That way, the generator will return boolean values, not the objects.